nipype/interfaces/niftyreg/regutils.py

```diff
@@ -326,6 +335,14 @@ class RegTools(NiftyRegCommand):
     output_spec = RegToolsOutputSpec
     _suffix = '_tools'
 
+    # Need this overload to properly constraint the interpolation type input
+    def _format_arg(self, name, spec, value):
+        if name == 'inter_val':
+            inter_val = {'NN': 0, 'LIN': 1, 'CUB': 3, 'SINC': 5}
```

Comment:
Can you go ahead and change `5` to `4` here and in `RegResample`?

Reply:
Sure